inference-engine/src/gna_plugin/gna2_model_debug_log.cpp

```diff
@@ -19,6 +19,9 @@
 #include <cmath>
 #include <map>
 
+#define unknownStr std::string {"unknown"}
+#define unknownVec std::vector<std::string> {{"unknown"}}
```

Comment:
avoid defines in C++
consider to use const/constexpr instead

Reply:
This is due to cpplint complaining about static const std::string - it requires C-string definition for static/global string constant.

Reply:
Consider template function (with specializations if needed) so could use (in `FindInMapOrReturnUnknown`):
GetUnknown<MapType::mapped_type>();

Reply:
Done.